ehrapy/io/_read.py

```diff
@@ -37,8 +39,9 @@ def read_csv(
         sep: Separator in the file; either , (default) or \t.
         index_column: The index column of obs. Usually the patient visit ID or the patient ID.
         columns_obs_only: These columns will be added to obs only and not X.
+        columns_x_only: These columns will be added to X only and all remaining columns to obs. Note that datetime columns will always be added to .obs though.
         return_mudata: Whether to create and return a MuData object. This is primarily used for complex datasets which require several AnnData files.
-        cache: Whether to use the cache when reading.
+        cache: Whether to write to cache when reading or not.
```

Comment:
Could add the default here -> 
```suggestion
        cache: Whether to write to cache when reading or not. (default: False)
```